css/syntaxes.json

```diff
@@ -11,12 +11,24 @@
   "animateable-feature": {
     "syntax": "scroll-position | contents | <custom-ident>"
   },
+  "any-value": {
+    "syntax": "[!]+"
```

Comment:
I don't know what means such syntax. Could you explain?

Reply:
Heh, I was guessing what it should possibly be, based on reading this defintion https://drafts.csswg.org/css-syntax-3/#typedef-any-value

Reply:
Oh, well, that' diffidently incorrect. `[!]+` means one or more exclamation marks and nothing else ;)
`<any-value>` can't be represented with such syntax, since it means any valid CSS value. Actually it's a generic and should be excluded from `syntax.json`.

Reply:
Thank you again ;-)

I have opened up a separate issue to track thisL: https://github.com/mdn/data/issues/190